packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -14,6 +14,46 @@ import 'package:vector_math/vector_math_64.dart';
 
 import 'debug.dart';
 
+/// Data collected during a layer hit test.
+class LayerHitTestEntry<T> {
+  /// Creates a hit test entry.
+  LayerHitTestEntry(this.value);
+
+  /// The [HitTestTarget] encountered during the hit test.
+  final T value;
+
+  @override
+  String toString() => '$value';
```

Comment:
prefer `'$runtimeType($value)'` otherwise it won't be clear what you're printing

Reply:
I agree. Although `HitTestEntry` is doing the same https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/gestures/hit_test.dart#L57

Reply:
we should fix that one too